Client/Frontend/Fakespot/Client/ProductAdsData.swift

```diff
@@ -10,7 +10,7 @@ struct ProductAdsData: Codable {
     let imageUrl: URL
     let price: String
     let currency: String
-    let grade: String
+    let grade: ReliabilityGrade
```

Comment:
What is the reason behind this change? Because of this the app is not building anymore.

Reply:
I'm not sure why it's not building for you, it didn't throw any error when I tested

Reply:
My mistake. It was in my test data. Sorry about that.